docs/strategy-customization.md

```diff
@@ -311,18 +311,18 @@ The name of the variable can be chosen at will, but should be prefixed with `cus
 ```python
 class AwesomeStrategy(IStrategy):
     # Create custom dictionary
-    cust_info = {}
+    custom_info = {}
```

Comment:
i guess it might be a good idea to move the whole "storing information" section to the advanced page?

i don't think you'll want to (or be able to) use this if you're just starting out ... so i'd consider it an advanced topic (we didn't have that section when this was documented i think) ?

Reply:
done